themes/minimal.scss

```diff
@@ -14,17 +14,21 @@ $navbarBgLight: $borderColor;
 $outlineColor: #b9b9b9;
```

Comment:
added .scss file ending. was it missing on purpose?

Reply:
don't know, is there a reason behind it @Jikstra?

Reply:
Nope dont think so